Assets/Scripts/PlayerController.cs

```diff
@@ -79,11 +88,87 @@ void GetGroundChecker()
         }
     }
 
+
     void CheckBottomBoundary()
     {
         if (groundChecker.IsTouchingLayers(LayerMask.GetMask("BottomBoundary"))) 
         {
             SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex);
         }
     }
+    #endregion
+
+    #region Size Shifting
+
+    void OnTitan(InputValue value)
+    {
+        Debug.Log("Titan");
+
+        titan.SetActive(true);
+
+        pixie.SetActive(false);
+        nomad.SetActive(false);
+
+        maxJumpCount = 0;
+    }
+
+    void OnNomad(InputValue value)
+    {
+        Debug.Log("Nomad");
+
+        nomad.SetActive(true);
+
+        pixie.SetActive(false);
+        titan.SetActive(false);
```

Comment:
To avoid issues with colliders when we add the animations please add the `SetActive(false);` functions first before activating the new G.O

Reply:
Done